doc/modules/model_evaluation.rst

```diff
@@ -1307,6 +1308,65 @@ Here is a small example of usage of this function:::
     <ftp://ftp.library.noaa.gov/docs.lib/htdocs/rescue/mwr/078/mwr-078-01-0001.pdf>`_,
     Monthly weather review 78.1 (1950)
 
+
+Calibration loss
+----------------
+
+The :func:`calibration_loss` function computes the expected and maximum
+calibration losses as defined in [1] for binary classes.
+
+Across all items in a set N predictions, the calibration loss measures
+the aggregated difference between (1) the average predicted probability
+assigned to the possible outcomes for item i, and (2) the frequencies
+of the actual outcome.
+Therefore, the lower the calibration loss is for a set of predictions, the
+better the predictions are calibrated.
+The aggregation method can be either:
+- 'sum' for :math:`\sum_k P_k \delta_k`, denoted as expected calibration error
```

Comment:
I haven't checked the rendering, but I am pretty sure you need blank lines before and after this list.